### ISSUE TITLE
Syntax error on tasks/conf/template-config.yml

### ISSUE THREAD

**Issue by langnix** [nginxinc/ansible-role-nginx#30]
There is a syntax-error in template-config.yml in the conditions:
`when: default_exists.stat.exists && main_template_enable`

when running with:
```
---
- hosts: localhost
  become: true
  roles:
    - role: nginxinc.nginx
  vars:
    - main_template_enable: true
```
this bounces with
```
."msg": "The conditional check 'default_exists.stat.exists && main_template_enable' failed. The error was: template error while templating string: unexpected char u'&' at 33
```

To fix: replace the '&&' with 'and'

**Comment by alessfg** [comment]
Hi @langnix! If I am not mistaken this issue was already reported in #20 and fixed in #21. Can you test the latest version of the role and check if it works?

**Comment by langnix** [comment]
Hi Alessandro, your absolutely right.. It's already fixed. Sadly the fix is not yet available on the galaxy (v0.2).😢

**Comment by alessfg** [comment]
Pushed a new release (v.0.3). You should now be able to see the changes reflected on Galaxy!
